ui/src/components/ItemList.svelte

```diff
@@ -34,7 +34,7 @@
 				on:click={() => {
 					SendNUI('hoverOut')
 
-					if (furniture.type) {
+                    if (furniture.type && furniture.max) {
```

Comment:
This is wrong. You should just check if theres a type. Because in a few lines after you set it to 1 if its null. 

I think you should just check if theres a type and if theres a max then do the `temsWithSameType.length >= furnitureTypeMax`

and if it doesnt have a max, just go through

Reply:
The problem is this line `let furnitureTypeMax = furniture.max || 1;` was originally assuming that all furnitures with a `type` would either define a max or it would default to 1. 

What we are now wanting to do is, say if I have a furniture with type defined ,then uses its max if defined, if not defined there is no limit. So essentially we just want to remove the default to 1. 

Make sure wardrobe and stash are defined as max 1 in the config.
Change the code on 41 to not default, then on line 43, we just first need to check if furniture.max is defined before checking length against it.

`if (furniture.max && itemsWithSameType.length >= furniture.max) {`